modules/ocf_www/manifests/site/www.pp

```diff
@@ -56,6 +69,8 @@
           '%{REQUEST_URI} !^/icons/',
           # ...hide ocfweb metrics
           '%{REQUEST_URI} !^/metrics',
+    # ...and not if it's the matrix well-known file
+    '%{REQUEST_URI} !^/\.well-known/matrix',
```

Comment:
The indentation looked right when you were in vim but now it doesn't? Estoy confundido.

Reply:
^^

Reply:
I think the linter did that? I wonder if I try to fix it if it will do it again